samples/client/petstore/dart/flutter_petstore/openapi/lib/model/user.dart

```diff
@@ -2,21 +2,21 @@ part of openapi.api;
 
 class User {
   
-  int id = null;
+  int id = ;
```

Comment:
Would this be an issue? Should it be just `int id;` ?

Reply:
That's also what I said in my comment

Reply:
ah ok, dart1 template is broken now :( sorry, haven't noticed it

Reply:
that would mean dart1 is not being tested but code generator is released and considered stable.

What's your approach to dead-code generators @wing328 ? 

Candidate for removal or improvement?